modules/integrate-with-qcr-scanner.adoc

```diff
@@ -8,15 +8,16 @@
 You can integrate {product-title} with Quay Container Registry for scanning images.
 
 .Prerequisites
-* You need an *OAuth Token* for authentication with the Quay Container Registry.
+* You must have an OAuth tokent for authentication with the Quay Container Registry.
```

Comment:
Typo for token.

Can you also share some record regarding the discussions for using the parens on the UI icons?  I wasn't a part of the discussion and other peer reviewers can benefit from knowing there is an exception for this document.